packages/docs/docs/foxie-the-fixie-fox.md

```diff
@@ -0,0 +1,12 @@
+import FoxieSidekick from '../static/img/foxie-sidekick.png'
+
+# Foxie (AKA "the Fixie fox")
+
+Meet Foxie. She is the Fixie mascot. Unlike wild foxes, Foxie doesn’t avoid humans…she looks to help them accomplish hard things.
+
+Foxie is highly intelligent, adaptable, and playful. In Chinese culture, foxes are believed to be able to shape-shift. Foxie represents
+this shape-shifting ability with her ability to seamlessly move across the “pro-code” to “no code” spectrum, making tools and
+capabilities that are delightful for everyone from professional software developers writing custom applications on top of LLMs
+to non-technical information workers who are looking to leverage the power of AI to get stuff done.
+
+<img src={FoxieSidekick} alt="Foxie, the Fixie mascot, doing a sidekick!" width="400"/>
```

Comment:
Github is being weird. I'm pretty sure you landed some of these changes yesterday:

Current changes:
![image](https://github.com/fixie-ai/ai-jsx/assets/5062458/406f74ff-1d2d-40f8-b3b4-1907c1f0027e)
Actual code in `main`:
![image](https://github.com/fixie-ai/ai-jsx/assets/5062458/4fa845db-72b8-4748-80f9-7888c963c1a7)
PR that already had this change:
![image](https://github.com/fixie-ai/ai-jsx/assets/5062458/79d11d7e-a82c-49e1-ad12-82c0c600ee59)